### PR TITLE
completed html-lists-Carolyne Golias

### PR DESCRIPTION
**Pull request by GCarolyne**

### CODE REVIEW

File: html-lists/html-lists-quiz-notes.md [hunk 1]
```diff
@@ -0,0 +1,37 @@
+# html-lists-quiz-notes
+
+## Quiz Questions
+
+Answer the following questions in the provided markdown file before turning in this exercise:
+
+- What is the difference between an ordered list and an unordered list in HTML?
+  A ordered list creates list where each item is numbered. An unordered list create lists where each item is marked with bullets or other similar markers. They both use the <li> .
+- Is an HTML list a block element or an inline element?
+
```
Comment (shawnkost) [inline html-lists/html-lists-quiz-notes.md]: Missing quiz question

**Review by shawnkost — CHANGES_REQUESTED** [review]
Missing a quiz question, and need to include a screenshot!

**Comment by GCarolyne** [comment]
Screenshot 2024-10-30 at 10 18 27 AM

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**